README.md

```diff
@@ -29,12 +29,29 @@ resources:
     private_key: |
             ...
 
+- name: sync-resource-multiple
+  type: rsync-resource
+  source:
+    servers:
+      - server1
+      - server2
```

Comment:
Also needs documentation stating that the first server in the list is the one used as the `check` and `in` origin